packages/create-cloudflare/src/templates.ts

```diff
@@ -11,7 +11,12 @@ import degit from "degit";
 import { C3_DEFAULTS } from "helpers/cli";
 import { readJSON, usesTypescript, writeJSON } from "helpers/files";
 import { validateTemplateUrl } from "./validators";
-import type { C3Args, C3Context } from "types";
+import type { C3Args, C3Context, PackageJson } from "types";
+
+export type PackageTransformer = (
+	pkgJson: PackageJson,
+	ctx: C3Context
+) => Promise<Record<string, string | object>>;
```

Comment:
is this used anywhere?

![Screenshot 2024-02-15 at 23 02 29](https://github.com/cloudflare/workers-sdk/assets/61631103/07a0504e-cbcd-4427-9b6f-3e0af6be6dac)

Reply:
Good catch. It was gonna be, but it ended up being worse than what we have.